metagraph/src/graph/annotated_dbg.hpp

```diff
@@ -122,6 +123,11 @@ class AnnotatedDBG : public AnnotatedSequenceGraph {
                               double presence_ratio,
                               const std::vector<double> &count_quantiles) const;
 
+    std::vector<std::pair<Label, std::vector<SmallVector<uint64_t>>>>
+    get_kmer_coordinates(std::string_view sequence,
```

Comment:
How about splitting this function into two: one which takes a sequence, and another which takes annotation indices (like the other methods)? It'll be more efficient to query if the nodes have already been mapped.

Reply:
done